src/main/java/org/fundacionjala/pivotal/api/APICommons.java

```diff
@@ -1,15 +1,21 @@
 package org.fundacionjala.pivotal.api;
 
 import com.jayway.restassured.response.Response;
+
+import java.util.ArrayList;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 /**
  * APICommons.java
  * Class with common method to execute an API request.
  */
 public final class APICommons {
-    private static Map<String, Response> requestResponse;
+    private static Map<String, Response> requestResponse = new HashMap<>();
+    private static List<Map<String, Response>> requestResponseList = new ArrayList<>();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "requestResponseList" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)